primitives/runtime/Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "sp-runtime"
-version = "4.0.0"
+version = "4.1.0-dev"
```

Comment:
Why did you bumped the version? 

Based on your change this is not correct anyway. Could you please revert this.

Reply:
Leaving `4.0.0` is incorrect either, the code has changed and the version in the codebase differs from what is on `crates.io`. I believe I saw similar changes for other crates recently when changes were introduced to already published crates.

Reply:
Yes, but then you would need to bump it to 5.0.0

Reply:
If this is considered a breaking change - yes. I can bump to `5.0.0-dev` if that makes more sense, I don't see specific guidelines for that in contribution doc.

Reply:
Similar bump was done in #10430 for instance. Let me know if you still want to revert version change here so we can move forward with this.

Reply:
Merge master and then we can continue.